cmd/kubectl-directpv/utils.go

```diff
@@ -127,3 +102,29 @@ func getInput(msg string) string {
 	fmt.Scanln(&input)
 	return input
 }
+
+func validateOutputFormat(isWideSupported bool) error {
+	switch outputFormat {
+	case "":
+	case "wide":
+		if !isWideSupported {
+			return errors.New("wide option is not supported by this command")
+		}
+		wideOutput = true
+	case "yaml":
+		yamlOutput = true
+	case "json":
+		jsonOutput = true
+	default:
+		return errors.New("--output flag value must be one of wide|json|yaml or empty")
+	}
+	printer = printYAML
+	if jsonOutput {
+		printer = printJSON
+	}
+	return nil
+}
+
+func dryRun() bool {
+	return dryRunFlag || yamlOutput || jsonOutput
```

Comment:
Why does output mode control dry-run? Many commands would perform actual change and print results as per output mode.

Reply:
`kubectl directpv install -o yaml` is a dry-run mode

Reply:
we have removed `--dry-run` from install command @balamurugana